lib/generators/rodauth/templates/app/views/rodauth/tailwind/change_login.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title, rodauth.change_login_page_title %>
+<div class="dark:bg-gray-900 flex items-center justify-center h-screen">
+  <div class="max-w-sm">
+    <h3 class="mb-3 text-3xl font-semibold text-center dark:text-gray-200"> Change Your Login Email Addres </h3>
+    <%= form_with url: rodauth.change_login_path, method: :post, data: { turbo: false } do |form| %>
+      <div class="mx-4 py-1">
+        <!-- The label should probably be named "email"  under rodauth's settings-->
+        <%= form.label "login", "Email", class: "block text-sm text-gray-400 dark:text-gray-100" %>
```

Comment:
We should use `rodauth.login_label` instead of "Email".

Reply:
Fixed: [86d1a93](https://github.com/janko/rodauth-rails/pull/114/commits/86d1a936ab82d0e1df32038dbb4f0624af54a5b7)

Reply:
I'll look out for similar instances too.